modules/rhcos-match-vulnerability.adoc

```diff
@@ -6,6 +6,6 @@
 = Vulnerability matching
 
 [role="_abstract"]
-Central services, which include Central and Scanner, perform vulnerability matching. Scanner uses Red{nbsp}Hat's Open Vulnerability and Assessment Language (OVAL) v2 security data streams to match vulnerabilities on {op-system-first} software components.
+Central services, which include Central and Scanner, perform vulnerability matching. The StackRox Scanner uses Red{nbsp}Hat's Open Vulnerability and Assessment Language (OVAL) v2 security data streams to match vulnerabilities on {op-system-first} software components. Scanner V4 uses https://security.access.redhat.com/data/csaf/v2/vex/[VEX].
 
-Unlike the earlier versions, {product-title-short} 4.0 no longer uses the Kubernetes node metadata to find the kernel and container runtime versions. Instead, it uses the installed {op-system} RPMs to assess that information.
+Unlike the earlier versions, {product-title-short} releases after 4.0 no longer use the Kubernetes node metadata to find the kernel and container runtime versions. Instead, {product-title-short} uses the installed {op-system} RPMs to assess that information.
```

Comment:
Just checking: do you think it's clear to users this is specifically for RHCOS nodes? Yes, it's on the RHCOS page, but knowing customers, they don't read 😂. Should we emphasize this is just for RHCOS?

Reply:
Added to the title, does that help?